pylzy/lzy/api/v1/__init__.py

```diff
@@ -235,43 +247,49 @@ def workflow(
         docker_image: Optional[str] = None,
         docker_pull_policy: DockerPullPolicy = DockerPullPolicy.IF_NOT_EXISTS,
         local_modules_path: Optional[Sequence[str]] = None,
-        provisioning: Provisioning = Provisioning.default(),
         interactive: bool = True,
-        cpu_type: Optional[str] = None,
-        cpu_count: Optional[int] = None,
-        gpu_type: Optional[str] = None,
-        gpu_count: Optional[int] = None,
-        ram_size_gb: Optional[int] = None,
+        provisioning: Optional[Provisioning] = None,
+        cpu_type: Optional[StringRequirement] = None,
```

Comment:
Why is default value `None` instead of `Any`?

Reply:
Because you can do:
```
with workflow(cpu_count=1):
    op(cpu_count=Any)
```

And from the other hand
```
with workflow(cpu_count=1):
    op()
    # or
    op(cpu_count=None)
```
should not override workflow's cpu_count value.